src/txn/txn_ckpt.c

```diff
@@ -2204,7 +2198,8 @@ __checkpoint_mark_skip(WT_SESSION_IMPL *session, WT_CKPT *ckptbase, bool force)
      */
     F_CLR(btree, WT_BTREE_SKIP_CKPT);
     if (!btree->modified && !force) {
-        deleted = 0;
+        int deleted = 0;
+        WT_CKPT *ckpt;
```

Comment:
Hi @sueloverso , thanks for reviewing the PR even before I'd reach out to reviewers!

Thanks for pointing that ordering. I put the declaration of `ckpt` at that point as it means it is declared the line before it is first used rather than the two being seperated. 

So my PR currently has:

```
    if (!btree->modified && !force) {
        int deleted = 0;
        WT_CKPT *ckpt;
        WT_CKPT_FOREACH (ckptbase, ckpt) {
```

and your suggestion is

```
    if (!btree->modified && !force) {
        WT_CKPT *ckpt;
        int deleted = 0;
        WT_CKPT_FOREACH (ckptbase, ckpt) {
```

Let me know which of the two options you prefer, and I'll go with it.
Thanks, Jeremy

Reply:
And the C coding standard states:

_All groups of things should be in alphabetical order where possible (local variable declarations, flags, stat fields, etc)_

and that text will remain unchanged in this PR.

Reply:
Hi @sue, that suggestion makes sense as I'm a fan of using layout to make code easier to read.

Reply:
I've updated that code to re-order the declarations and add a blank line.